_events/2020/2020-07-pearc20.md

```diff
@@ -12,19 +12,63 @@ repeated: false
+The **Research Software Engineers Community Workshop** will be July 31st, starting a 8am PDT (Portland, OR time).  It is open to all [PEARC20](https://pearc.acm.org/pearc20/) registrants.  
 
-The **Research Software Engineers Community Workshop** will be July 31st, starting a 8am PDT (Portland, OR time).  It is open to all PEARC20 registrants.  
+The full PEARC20 conference runs July 26-31 and is virtual this year.  Information about the conference is available on the [PEARC20
+conference website](https://pearc.acm.org/pearc20/).  [PEARC20 Registration](http://www.cvent.com/events/practice-and-experience-in-advanced-research-computing-2020-pearc-/event-summary-e4cda8bfe4c545b7bb3e458e13a128c2.aspx?tw=12-7B-8D-67-26-C9-CC-DD-AD-82-E2-F8-E0-D3-05-97) will be required for all workshop participants.
 
 The workshop will provide an opportunity for
 research software engineers to connect and share their work.  There will be
 presentations from RSEs about their software, research, projects, and groups, and about
 the development of the RSE community.  There will also be discussion of challenges facing RSEs and what the community can do to help meet those challenges.
 
-Short talks will be solicited from the community soon.
+You can participate in the workshop by giving a short talk or joining as a general participant.  To give a talk, please see the guidelines below.  To join as a participant, [register for PEARC20](http://www.cvent.com/events/practice-and-experience-in-advanced-research-computing-2020-pearc-/event-summary-e4cda8bfe4c545b7bb3e458e13a128c2.aspx?tw=12-7B-8D-67-26-C9-CC-DD-AD-82-E2-F8-E0-D3-05-97) and then join our workshop session on Friday.
+
+### Propose a Talk
+
+Are you a research software engineer?  Are you a researcher or student writing software?  Are you a software developer writing software for research?  We want to hear and learn from you at the workshop.
+
+Possible talk topics:
+
+* How you solved a problem on a project
+* The research impact of software you helped create
+* How software you helped develop could help other RSEs or researchers
+* A framework or package that has helped you solve a problem
+* A challenge or opportunity for research software in a particular research field
+* How research software engineers are organized at your institution
+* Teaching software engineering skills to others
+* Your RSE career path
+* Opportunities and challenges for research software and RSEs overall (sustainability, culture, citation)
+
+Talks from all research fields and concerning all types of technology are welcome. Group submissions with multiple authors are encouraged, however we ask that you designate only one author as presenter.  
+
+Proposals from individuals who may be first time conference presenters or consider themselves to be part of an underrepresented group in the research software engineering community are particularly encouraged.  
+
```

Comment:
It's probably implied that talk == join into a virtual meeting / Zoom and then share slides, but maybe we should state that explicitly (e.g., you could imagine recording in advance, or creating a presentation different from standard slides).